internal/exec/util/passwd.go

```diff
@@ -22,12 +22,21 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/coreos/go-systemd/v22/journal"
 	"github.com/coreos/ignition/v2/config/v3_1_experimental/types"
 	"github.com/coreos/ignition/v2/internal/as_user"
 	"github.com/coreos/ignition/v2/internal/distro"
 	"golang.org/x/sys/unix"
 )
 
+const (
+	// ignitionSSHAuthorizedkeysMessageID keeps track of the journald
+	// log related to ssh_authorized_keys information.
+	ignitionSSHAuthorizedkeysMessageID = "225067b87bbd4a0cb6ab151f82fa364b"
+)
+
+var ignitionSSHAuthKeysNum uint
```

Comment:
I don't think we want this to be a global variable, should be a local in `AuthorizeSSHKeys` right?

Reply:
if we don't need to worry about printing out the number of keys that were added then we shouldn't need this variable at all, right?

Reply:
Yep, agreed.  We probably don't need the number of keys.